### DISCUSSION TITLE
Query Builder

### DISCUSSION THREAD

**Discussion by acharest2020** [dbeaver/dbeaver#12122] [General]
When you use the query builder at the bottom there is a section where you can put in your filters.  I will also show you what fields you have selected.  On the right hand side, you can see the sql statement.  This has disappeared.  How do you get it back?

**Comment by CairoKolton** [comment]
I am new to Mysql and SQL in general. Im looking to take a course to learn it but all the udemy and other courses I can find use workbench and/or just code. [essay writer](https://essaywriter.fun/) Any suggestions of a good course online which teaches mySQL using DBeaver? Even a tutorial on creating a basic database w Dbeaver would probably suffice.

**Reply by LonwoLonwo** [reply]
Unfortunately, I can not advise you a good course. But we tried to make DBeaver user-friendly. 
So you can in practice try creating databases, tables, columns and record data via UI.
Also we have documentation, which can be useful to you: https://github.com/dbeaver/dbeaver/wiki

**Comment by LonwoLonwo** [comment]
Hello @acharest2020 
![2021-04-20 10_36_29-DBeaver Enterprise 21 0 3 - _orcl_ Script-42](https://user-images.githubusercontent.com/45152336/115356583-7398fc80-a1c4-11eb-9313-fc7e003122ec.png)

![2021-04-20 10_37_00-DBeaver Enterprise 21 0 3 - _orcl_ Script-42](https://user-images.githubusercontent.com/45152336/115356586-74319300-a1c4-11eb-81a9-4d57a0a39507.png)

You can use button with monitor image or Ctrl+Schift+B in Query Builder.

Did it help you?
